src/Components/IssuesControl.test.jsx

```diff
@@ -1,57 +1,57 @@
 import React from 'react'
-import {act, render, renderHook, screen} from '@testing-library/react'
+import {act, render, renderHook} from '@testing-library/react'
 import ShareMock from '../ShareMock'
 import useStore from '../store/useStore'
 import {IssuesNavBar, Issues} from './IssuesControl'
 
 
 test('Issues NavBar Issues', () => {
-  render(<ShareMock><IssuesNavBar/></ShareMock>)
-  expect(screen.getByText('Notes')).toBeInTheDocument()
+  const {getByText} = render(<ShareMock><IssuesNavBar/></ShareMock>)
+  expect(getByText('Notes')).toBeInTheDocument()
 })
 
 
-test('Issues NavBar Comments', () => {
+test('Issues NavBar Comments', async () => {
   const {result} = renderHook(() => useStore((state) => state))
   const testIssueId = 10
-  act(() => {
+  const {getByTitle} = render(<ShareMock><IssuesNavBar/></ShareMock>)
+  await act(() => {
     result.current.setSelectedIssueId(testIssueId)
   })
-  render(<ShareMock><IssuesNavBar/></ShareMock>)
-  expect(screen.getByTitle('Back to the list')).toBeInTheDocument()
+  expect(await getByTitle('Back to the list')).toBeInTheDocument()
 })
 
 
-test('Issues ', () => {
+test('Setting issues in zustand', async () => {
   const {result} = renderHook(() => useStore((state) => state))
-  act(() => {
+  const {getByText} = render(<ShareMock><Issues/></ShareMock>)
+  await act(() => {
     result.current.setSelectedIssueId(null)
   })
-  act(() => {
+  await act(() => {
     result.current.setIssues(MOCK_ISSUES)
   })
-  render(<ShareMock><Issues/></ShareMock>)
-  expect(screen.getByText('open_workspace')).toBeInTheDocument()
-  expect(screen.getByText('closed_system')).toBeInTheDocument()
+  expect(await getByText('open_workspace')).toBeInTheDocument()
+  expect(await getByText('closed_system')).toBeInTheDocument()
 })
 
 
-test('Issues ', () => {
+test('Setting comments in zustand ', async () => {
   const {result} = renderHook(() => useStore((state) => state))
   const testIssueId = 10
-  act(() => {
+  const {getByText} = render(<ShareMock><Issues/></ShareMock>)
+  await act(() => {
```

Comment:
@pablo-mayrgundter this worked to resolve the warning

Reply:
Heya, just getting to this.  Thanks but also needs a merge now.  Ogali has a PR for just the await fix that will probably be in soon too.

Reply:
I think Ogali merged his fix.
there are conflicts now. 
shall I rebase?

Reply:
Yes sure